### PR TITLE
Sorting

### PR DESCRIPTION
**Pull request by ThunderFlight**

### CODE REVIEW

File: src/main.zig [hunk 1]
```diff
@@ -47,10 +48,11 @@ pub fn main() !void {
 
     var playlist_paths: [2][]const u8 = .{
         try std.fs.path.join(allocator, &[2][]const u8{ home.?, ".config/cmus/playlists" }),
-        "/home/vktrenokh/.config/cmus/xplaylists/",
+        "/home/cosmo/.config/cmus/xplaylists/",
```
Comment (VKTRenokh) [inline src/main.zig]: Remove this line.

File: src/main.zig [hunk 2]
```diff
@@ -17,7 +18,7 @@ const Event = union(enum) {
 
 pub fn main() !void {
     var arena = std.heap.ArenaAllocator.init(std.heap.page_allocator);
-    const allocator = arena.allocator();
+    var allocator = arena.allocator();
```
Comment (VKTRenokh) [inline src/main.zig]: why var?

File: src/sorting.zig [hunk 3]
```diff
@@ -0,0 +1,24 @@
+const std = @import("std");
+const playlists = @import("playlists.zig");
+
+pub const SortTypes = enum {
```
Comment (VKTRenokh) [inline src/sorting.zig]: Rename this enum to SortMethods please.

File: src/sorting.zig [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    less,
+    greater,
+};
+
+pub fn sort(list: std.ArrayList(*playlists.Playlist), sortType: SortTypes) !void {
+    const sorted_list = switch (sortType) {
+        .less => std.mem.sort(*playlists.Playlist, list.items[0..], {}, lessThan),
+        .greater => std.mem.sort(*playlists.Playlist, list.items[0..], {}, greaterThan),
+    };
```
Comment (VKTRenokh) [inline src/sorting.zig]: Remove repeating code please.

File: src/sorting.zig [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+    std.debug.print("{any}\n", .{sorted_list});
```
Comment (VKTRenokh) [inline src/sorting.zig]: remove this line please

**Review by VKTRenokh — CHANGES_REQUESTED** [review]

File: src/sorting.zig [hunk 6]
```diff
@@ -1,18 +1,21 @@
...
+pub const SortMethods = enum {
     less,
     greater,
 };
 
-pub fn sort(list: std.ArrayList(*playlists.Playlist), sortType: SortTypes) !void {
-    const sorted_list = switch (sortType) {
-        .less => std.mem.sort(*playlists.Playlist, list.items[0..], {}, lessThan),
-        .greater => std.mem.sort(*playlists.Playlist, list.items[0..], {}, greaterThan),
+pub fn chooseSort(list: std.ArrayList(*playlists.Playlist), sortType: SortMethods) !void {
```
Comment (VKTRenokh) [inline src/sorting.zig]: parameters should be named in snake_case

Comment (VKTRenokh) [inline src/sorting.zig]: Rename this fn pls D:

**Review by VKTRenokh — CHANGES_REQUESTED** [review]

**Review by VKTRenokh — APPROVED** [review]

**Review by SlikeDollar — APPROVED** [review]

**Pull request merged**